src/cf.data.pre

```diff
@@ -2251,6 +2238,23 @@ DOC_START
 			reference a PEM file containing both the certificate
 			and private key.
 
+	   generate-host-certificates=[off|FILE]
+			Dynamically create X.509 server certificates for the
+			destination hosts of received (or bumped) TLS traffic.
+
+			FILE is the path to file containing an X.509 CA
+			certificate and private key in PEM format, to be used
+			for signing certificates generated for the TLS
+			handshake ServerHello.
+
+			If there is a CA certificate, lifetime of the generated
+			certificate equals lifetime of the CA certificate. If
+			generated certificate is self-signed lifetime is three
+			years.
+
+			This option is enabled by default when Squid is built
```

Comment:
I believe this behaviour is what is already happening. Updated the docs to match.

Reply:
> I believe this behaviour is what is already happening.

It certainly was _not_ happening when I made that change request, but I am glad you fixed the code (assuming you have).

> Updated the docs to match.

The updated docs say `this option defaults to enabled`. This change request was, in part, about removing that kind of phrasing (items 1 and 2 on the first list). It is still there. To fix this, I recommend describing Squid actions (when the option is not specified) rather than saying that the option is "enabled by default" or "defaults to enabled" (when the option is not specified).